tox.ini

```diff
@@ -77,7 +77,9 @@ changedir = {toxinidir}
 deps =
     ansible
     colorama
+    mock
```

Comment:
please put the extra requirements in the pylint_extra_requirements.txt file - the system roles are a little bit  special since they try to use a common tox.ini from the template repository in the system roles project.